drink/src/session/mock/extension.rs

```diff
@@ -1,24 +1,26 @@
 use std::sync::{Arc, Mutex};
 
+use ink_sandbox::pallet_revive::ExecReturnValue;
 use parity_scale_codec::{Decode, Encode};
 
 use crate::{
     errors::MessageResult,
-    pallet_contracts::{chain_extension::ReturnFlags, debug::ExecResult, ExecReturnValue},
-    pallet_contracts_debugging::InterceptingExtT,
+    pallet_revive::chain_extension::ReturnFlags,
     session::mock::{MockRegistry, Selector},
 };
 
+#[allow(dead_code)]
+#[deprecated = "No longer used due to pallet-revive removed `type Debug` for intercepting contract calls"]
 /// Runtime extension enabling contract call interception.
-pub(crate) struct MockingExtension<AccountId: Ord> {
+pub(crate) struct MockingExtension {
```

Comment:
AFAICT, this is not part of the public interface, so a deprecation warnings seems unnecessary.

Probably better to just comment this out and add a TODO to replace it with an equivalent API when we figure that out.